app/models/concerns/metric_types.rb

```diff
@@ -18,6 +18,10 @@ module MetricTypes
     ],
     'subject.first-to-classify' => [
       'classification_created'
+     ],
+    'classifier.first-to-classify.visible' => [
```

Comment:
It's an a/b split for a banner that notifies the user that they are first to classify the current subject. I tried to follow the convention Parrish seemed to be going with documented here: https://github.com/zooniverse/Seven-Ten/blob/master/docs/variants.md#attributes

The banner is in the classifier, the split is for first to classify banner, and it's a visibility split. I'm open to a more clear naming if someone can come up with one that seems to follow that convention already in place.